data-prepper-plugins/failures-common/src/main/java/org/opensearch/dataprepper/plugins/dlq/s3/S3DlqWriter.java

```diff
@@ -102,6 +105,7 @@ public void write(final List<DlqObject> dlqObjects, final String pipelineName, f
     private void doWrite(final List<DlqObject> dlqObjects, final String pipelineName, final String pluginId) throws IOException {
         final PutObjectRequest putObjectRequest = PutObjectRequest.builder()
             .bucket(bucket)
+            .expectedBucketOwner(bucketOwner)
```

Comment:
Is it backward compatible if bucketOwner is none?

Reply:
Yes you can pass null here and it won't apply a bucket owner